### ISSUE TITLE
Build fails if `rand` feature is disabled

### ISSUE THREAD

**Issue by tarcieri** [dalek-cryptography/ed25519-dalek#108]
```
   Compiling ed25519-dalek v1.0.0-pre.3
error[E0432]: unresolved import `rand`
  --> /Users/bascule/.cargo/registry/src/github.com-1ecc6299db9ec823/ed25519-dalek-1.0.0-pre.3/src/ed25519.rs:14:5
   |
14 | use rand::{CryptoRng, RngCore};
   |     ^^^^ use of undeclared type or module `rand`
```

[That line](https://github.com/dalek-cryptography/ed25519-dalek/blob/ee67f36/src/ed25519.rs#L14) needs a `#[cfg(feature = "rand")]` above it.

**Comment by isislovecruft** [comment]
Ah, oops, thanks @tarcieri!

**Comment by isislovecruft** [comment]
Fixed in 8a2e9af
